demos/demo5/Demo.jsx

```diff
@@ -18,6 +18,17 @@ const Demo = React.createClass({
     };
   },
 
+  componentWillMount() {
+    document.addEventListener('mousemove', this.handleMouseMove);
```

Comment:
What would be the expected behavior of `mouseLeave`? I can just scale the main div to the screen size